.gitignore

```diff
@@ -61,3 +61,5 @@ typings/
 
 # next.js build output
 .next
+.prettierrc
```

Comment:
Actually @hector0513 could you actually commit this file to GIT, so that you can us can all share formatters? 

Pegah and I will install and use Prettier as well